Dockerfile

```diff
@@ -81,16 +74,20 @@ ENV JAVA_8_HOME=/usr/lib/jvm/openjdk8
 ENV JAVA_11_HOME=/usr/lib/jvm/openjdk11
 ENV JAVA_17_HOME=/usr/lib/jvm/openjdk17
 
+ENV JAVA_ZULU7_HOME=/usr/lib/jvm/zulu7
```

Comment:
Do we still need JDK7 ?

Reply:
I think we still need as we test the fact the agent does not cash on Java 7 but prompts a nicely comment instead.

Reply:
Exactly. It's required for the `test-published-dependencies` job.